### DISCUSSION TITLE
wbeditentity now support adding statements to senses

### DISCUSSION THREAD

**Discussion by dpriskorn** [LeMyst/WikibaseIntegrator#146] [General]
see https://phabricator.wikimedia.org/T199896
Does this affect WBI?

**Comment by LeMyst** [comment]
Hello @dpriskorn ,

With WikibaseIntegrator, did you successfully edit Senses and Forms on Lexeme ?
I don't talk about claims, only adding/editing/removing senses/forms on lexemes

I never really looked the Lexeme part in WBI

**Comment by LeMyst** [comment]
Check #148
